### DISCUSSION TITLE
unable to change default language to english

### DISCUSSION THREAD

**Discussion by rusheedroberts** [snipe/snipe-it#13261] [General]
![image](https://github.com/snipe/snipe-it/assets/77894940/35f56722-5be9-42ee-b7c3-93b3ed6f6e4e)
![image](https://github.com/snipe/snipe-it/assets/77894940/1d4acf74-cb92-4140-8303-b5d20afc2f55)

**Comment by snipe** [comment]
Check your profile settings. There are three layers of language support. https://snipe-it.readme.io/docs/localization

**Reply by rusheedroberts** [reply]
hi, i found where the issue was. in the database for that user, it was set to German..
